Mathlib/Data/Fin/VecNotation.lean

```diff
@@ -0,0 +1,583 @@
+/-
+Copyright (c) 2020 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen
+
+! This file was ported from Lean 3 source module data.fin.vec_notation
+! leanprover-community/mathlib commit 2445c98ae4b87eabebdde552593519b9b6dc350c
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.Fin.Tuple.Basic
+import Mathlib.Data.List.Range
+import Mathlib.GroupTheory.GroupAction.Pi
+-- import Mathlib.Meta.Univs -- Porting note: missing
```

Comment:
I think this can just be removed, it's part of the `reflect` machinery that is somewhat redundant in light of `Q`.